libevm/lib/service_evm.go

```diff
@@ -24,14 +33,16 @@ type EvmContext struct {
 
 type EvmParams struct {
 	HandleParams
-	From       common.Address   `json:"from"`
-	To         *common.Address  `json:"to"`
-	Value      *hexutil.Big     `json:"value"`
-	Input      []byte           `json:"input"`
-	GasLimit   hexutil.Uint64   `json:"gasLimit"`
-	GasPrice   *hexutil.Big     `json:"gasPrice"`
-	AccessList types.AccessList `json:"accessList"`
-	Context    EvmContext       `json:"context"`
+	From          common.Address   `json:"from"`
+	To            *common.Address  `json:"to"`
+	Value         *hexutil.Big     `json:"value"`
+	Input         []byte           `json:"input"`
+	Gas           hexutil.Uint64   `json:"gas"`
```

Comment:
You renamed this on the Java side to `availableGas`, need to change this here too otherwise it always use `MaxInt64`.